include/openmc/tallies/tally.h

```diff
@@ -55,6 +57,8 @@ class Tally {
 
   int32_t n_filter_bins() const {return n_filter_bins_;}
 
+  bool get_writable() const { return writable_;}
```

Comment:
For getters, we typically omit the `get_` part, so this would just be
```suggestion
  bool writable() const { return writable_;}
```